packages/components/src/components/Note/Note.module.css

```diff
@@ -0,0 +1,71 @@
+@import "@/styles";
+
+.root {
+  border-radius: var(--note--border-radius);
+  border: var(--note--border-width--default) var(--note--border-style)
+    var(--note--info-border-color);
+  border-inline-start-width: var(--note--border-width--main);
+  padding: var(--note--padding);
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-column-gap: var(--note--icon-to-text-spacing);
+  grid-row-gap: var(--note--text-to-text-spacing);
+
+  /* Elements */
+
+  .customIcon,
```

Comment:
Die Verschachtelung der Klassen in die `root`-Klasse ist nicht notwendig. Lieber rausziehen. Macht es übersichtlicher.